src/styles/blocks/explore-online.scss

```diff
@@ -0,0 +1,69 @@
+.explore-online {
+  @include onTablet {
+    max-width: 536px;
+  }
+
+  @include onDesktop {
+    max-width: none;
+  }
+
+  &__title {
+    margin-bottom: 40px;
+  }
+
+  &__performance-wrapper {
+    @include onDesktop {
+      display: flex;
+    }
+  }
+
+  &__performance {
+    margin-bottom: 48px;
+    &:last-child {
```

Comment:
It's better to use :last-of-type instead of :last-child to target the last element with the same class in the container.